.github/workflows/test-demo.yaml

```
@@ -33,10 +33,15 @@ jobs:
         run: nix build .#overview
 
       - name: Run and test VM
+        env:
+          DISTRO: ${{ matrix.distro }}
+          NIX_CONFIG: |
+            extra-substituters = https://ngi.cachix.org/
+            extra-trusted-public-keys = ngi.cachix.org-1:n+CAL72ROC3qQuLxIHpV+Tw5t42WhXmMhprAGkRSrOw=
         run: >
           docker run
           --privileged
           --volume ./result/project/Cryptpad/default.nix:/default.nix
           --volume "$(pwd)/.github/workflows/test-demo.sh:/test-demo.sh"
```

The env var will have to be passed to docker as well:

```suggestion
          --volume "$(pwd)/.github/workflows/test-demo.sh:/test-demo.sh"
          -e DISTRO="$DISTRO"
```